---
title: "A dive into the coral microbiome"
author: "Hadrien Gourl√©"
date: "May 29, 2020"
output:
    revealjs::revealjs_presentation:
        theme: sky
        highlight: pygments
        center: true
        transition: slide
        template: assets/templates/template.html
        css: assets/css/custom.css
        reveal_options:
            slideNumber: true
---

```{r include=FALSE}
library(knitr)
```


# {#intro-slide data-background="assets/img/background.jpg"}

## {#intro-slide data-background="assets/img/background.jpg"}

<div id="intro-text-box">Coral reefs are one of the most diverse ecosystem on the planet.</div><br>
<div id="intro-text-box" class="fragment fade-left">They are economically vital for many people worldwide</div><br>
<div id="intro-text-box" class="fragment fade-left">They face great challenges and their population is rapidely declining.</div>

# The coral microbiome {#coral data-transition="zoom-out"}

![](assets/img/coral.svg)

# {data-transition="zoom-in"}

![](assets/img/microbes.svg)

# Technologies to study the microbiome

## Sequencing

![](assets/img/sequencing_metagenomes.svg)

## Bioinformatics

![](assets/img/intro_bioinformatics.svg)

# Why study the coral microbiome?

## {data-background="assets/img/why_bg.svg"}

# Aims

- Improve methodology within the field of metagenomics
- Understanding coral-associated microbes better

# Simulating Illumina Metagenomic Data with InSilicoSeq {#paper data-background="assets/img/paper_i.svg"}

## Why? {#paper data-background="assets/img/paper_i.svg"}

<div class="fragment fade-in">
why do we need simulated data?
</div>

## {#paper data-background="assets/img/paper_i.svg"}

![](assets/img/wordcloud.svg)

## Why? {#paper data-background="assets/img/paper_i.svg"}

why do we need simulated data?
<div class="fragment fade-in">
why do we need *yet another* simulation software?
</div>

## Results {#paper data-background="assets/img/paper_i.svg"}

Easily installable and usable

```bash
$ conda install insilicoseq
$ iss generate --ncbi bacteria -U 100 -n 10M \
    --model novaseq --cpus 8 --output reads
```

And used in the development of 10 new published metagenomic software.

## Results {#paper data-background="assets/img/paper_i.svg"}

Kernel Density Estimation

:::::: {style="display: flex;"}
::: {.column width="50%"}
![](assets/img/kde.svg)
:::
::: {.column width="50%"}
![](assets/img/kde_2d.svg)
:::
::::::

## Results {#paper data-background="assets/img/paper_i.svg"}

![](assets/img/qual.svg)

## {#paper data-background="assets/img/paper_i.svg"}

![](assets/img/wordcloud.svg)

# Paper II - A Data Driven Review of the Coral Microbiome {#paper data-background="assets/img/paper_ii.svg"}

## The 16S gene and metabarcoding {#paper data-background="assets/img/paper_ii.svg"}

![](assets/img/16S_regions.svg)

## Methods {#paper data-background="assets/img/paper_ii.svg"}

![](assets/img/paper_ii_workflow.svg)

## Methods {#paper data-background="assets/img/paper_ii.svg"}

- Dada2 handles fine-scale variations
- Less impact from contaminants

## Methods {#paper data-background="assets/img/paper_ii.svg"}

- Linear modelling with DESeq2
- GLM for each taxonomy assignment at the genus level

## Results {#paper data-background="assets/img/paper_ii.svg"}

:::::: {style="display: flex;"}
::: {.column width="40%" height="400px"}
![](assets/img/paper_ii_phyla.svg)
:::
::: {.column width="60%" height="400px"}
![](assets/img/paper_ii_proteo.svg)
:::
::::::

## Results {#paper data-background="assets/img/paper_ii.svg"}

![](assets/img/paper_ii_bleached.svg)

# Molecular Ecology of Coral Reef Microorganisms in the Western Indian Ocean coast of Kenya {#paper3 data-background="assets/img/paper_iii.svg"}

## {#paper3 data-background="assets/img/why_bg.svg"}

## Methods {#paper3 .workflow data-background="assets/img/paper_iii.svg"}

![](assets/img/paper_iii_workflow.svg)

## Results {#paper3 data-background="assets/img/paper_iii.svg"}

- 193 draft genomes recovered
- 12 million unique putative proteins

## Results {#paper3 .bar data-background="assets/img/paper_iii.svg"}

![](assets/img/paper_iii_genera.svg)

## Results {#paper3 .tree data-background="assets/img/paper_iii.svg"}

![](assets/img/tree_bact.svg)

## Results  {#paper3 data-background="assets/img/paper_iii.svg"}

![](assets/img/kegg.svg)

# Conclusion

- Warm water coral microbiomes are dominated by *Endozoicomonas*, but we are unsure what its role is
- Metagenomics allows insight into function, allowing to *dive* in the microbiome
- coral bacteria are sensitive to pollution, and are promising potential markers for monitoring reef health and eventually inform policy making

# Future perspectives

## Methods

- need for benchmarking is still there
- automatic metagenome binning methods have a lot of room for improvement

## Corals

- Whole metagenome sequencing on coral samples (SML)
- This project is ongoing, sampling and sequencing done!

# {#thanks data-background="assets/img/thank_you.svg"}
